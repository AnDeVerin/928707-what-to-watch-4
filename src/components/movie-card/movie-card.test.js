import React from 'react';
import renderer from 'react-test-renderer';
import MovieCard from './movie-card.jsx';

const movie = {
  title: 'Aviator',
  thumbUrl: 'img/aviator.jpg',
};

describe('MovieCard component', () => {
  it('renders correctly', () => {
    const component = renderer
      .create(
        <MovieCard movie={movie} onHover={jest.fn()} onClick={jest.fn()} />
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
