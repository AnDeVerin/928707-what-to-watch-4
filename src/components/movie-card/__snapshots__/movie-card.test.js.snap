// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard component renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
    onClick={[Function]}
  >
    <img
      alt="Aviator"
      height="175"
      src="img/aviator.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Aviator
    </a>
  </h3>
</article>
`;
